win32ss/user/ntuser/winpos.c

```diff
@@ -1743,6 +1745,41 @@ ForceNCPaintErase(PWND Wnd, HRGN hRgn, PREGION pRgn)
    }
 }
 
+static VOID FASTCALL IntImeWindowPosChanged(VOID)
+{
+    HWND hwndNode, *phwnd;
+    PWND pwndNode, pwndDesktop = UserGetDesktopWindow();
+    PWINDOWLIST pWL;
+    USER_REFERENCE_ENTRY Ref;
+
+    if (!pwndDesktop)
+        return;
+
+    /* Enumerate the windows to get the IME windows (of default and non-default) */
+    pWL = IntBuildHwndList(pwndDesktop->spwndChild, IACE_LIST, gptiCurrent);
```

Comment:
There was a patch that excludes IME windows from `IntBuildHwndList`, is this going to be a problem here?

Reply:
https://github.com/reactos/reactos/commit/0fee1fc80c07b0ea42d6fc28077c5081fde727d8
Is the patch this? `IntWinListOwnedPopups` and `IntBuildHwndList` are different.

https://github.com/reactos/reactos/blob/b2cac5cd6ca3c0491c47ab551de00e75b87ca2c1/win32ss/user/ntuser/window.c#L1422

Reply:
Indeed I misremembered, sorry.